dropwizard-core/src/main/java/io/dropwizard/setup/Bootstrap.java

```diff
@@ -86,10 +90,22 @@ public void registerMetrics() {
         getMetricRegistry().register("jvm.memory", new MemoryUsageGaugeSet());
         getMetricRegistry().register("jvm.threads", new ThreadStatesGaugeSet());
 
-        JmxReporter.forRegistry(metricRegistry).build().start();
+        jmxReporter = JmxReporter.forRegistry(metricRegistry).build();
+        jmxReporter.start();
+
         metricsAreRegistered = true;
     }
 
+    /**
+     * Returns the {@link JmxReporter} registered with the bootstrap's {@link MetricRegistry}.
+     *
+     * @since 2.0
```

Comment:
Should this be 2.0.2 or 2.1.0 (not sure which release it is planned to go out with)?

Reply:
Good point. Since it's an additional feature (API-wise), it should go into Dropwizard 2.1.0. 👍